src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-email-address.cypress.spec.js

```diff
@@ -0,0 +1,56 @@
+import { PROFILE_PATHS } from '../../constants';
+import mockUser from '../fixtures/users/user-36.json';
+import mockFeatureToggles from '../fixtures/feature-toggles.json';
+
+const setup = (mobile = false) => {
+  window.localStorage.setItem(
+    'DISMISSED_ANNOUNCEMENTS',
+    JSON.stringify(['single-sign-on-intro']),
+  );
+
+  if (mobile) {
+    cy.viewport('iphone-4');
+  }
+
+  cy.login(mockUser);
+  cy.route('GET', '/v0/feature_toggles*', mockFeatureToggles);
```

Comment:
Do you know if we have a ticket to clean up the profile 2.0 feature flag? Remove these lines in our tests could be part of that work.